spec/api_spec.rb

```diff
@@ -125,5 +159,19 @@ def app
 
       expect(content['status']).to eq('done')
     end
+
```

Comment:
This entire test shouldn't be here at all; it is a copy of a previous one. Please delete this test `can perform a build`.